src/tests/playwright/pages/util/recorder/recorder-main.js

```diff
@@ -17,33 +17,43 @@ export const record = async (context, recordLength) => {
   console.assert(context instanceof AudioContext);
   console.assert(typeof recordLength === 'number' && recordLength > 0);
 
-  const maxSamples = recordLength * context.sampleRate;
-
-  await context.audioWorklet.addModule('./util/recorder/recorder-processor.js');
-
-  const recorder = new AudioWorkletNode(context, 'recorder', {
-    processorOptions: {
-      maxSamples,
-    },
-  });
-
-  let bufferResolve;
-  recorder.port.onmessage = (e) => {
-    if (e.data.message === 'RECORD_DONE') {
-      const channelBuffers = e.data.buffer;
+  const recorderBufferSize = recordLength * context.sampleRate;
+
+  let recorder;
+
+  try {
```

Comment:
Hah! This is clever. :)

However, there might be other reasons that the construction of an AudioWorkletNode can fail. We don't need to fix this in this CL, but we might need a simpler gating mechanism.